.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -2,19 +2,23 @@
 name: "\U0001F680 Feature request"
 about: Suggest an idea for this project
 title: ''
-labels: 'Community: Request :hand:, Triage :white_flag:'
+labels: 'Community: Request :hand:'
 assignees: ''
 ---
 
-> :hand: Many people requests features. Tell us why yours is important to the
+<!-- ⚠️⚠️ Do Not Delete This! feature_request_template ⚠️⚠️ -->
+<!-- Please read our Rules of Conduct: https://github.com/OHIF/Viewers/blob/master/CODE_OF_CONDUCT.md -->
+
+<!-- :hand: Many people requests features. Tell us why yours is important to the
 > community. How does it add value? Why _this feature_?
 >
 > Is your request very specific to your needs? Consider
-> [contributing it](https://docs.ohif.org/contributing.html) yourself! Or reach
-> out to a community member that offers
+> [contributing it](https://v3-docs.ohif.org/development/contributing) yourself!
+> Or reach out to a community member that offers
 > [consulting services](https://docs.ohif.org/help.html#paid--commercial).
+-->
 
-## Request
+### Request
 
 **What feature or change would you like to see made?**
```

Comment:
@sedghi  consider  adding information like the following, not a big deal, just something to consider.

- Please include as much detail as possible including possibly mock up screen shots, workflow or logic flow diagrams etc.
- Discuss if and how the requested feature interacts with existing features